packages/code-studio/package.json

```diff
@@ -70,19 +68,21 @@
   ],
   "scripts": {
     "analyze": "source-map-explorer 'build/static/js/*.js'",
-    "start": "react-scripts --max-old-space-size=4096 start",
-    "build-js": "react-scripts --max-old-space-size=4096 build",
-    "build": "npm-run-all build-js clean-mocks",
-    "clean-mocks": "rm -rf build/__mocks__",
-    "eject": "react-scripts eject",
+    "start": "vite",
+    "build": "vite build",
+    "preview": "vite preview",
     "update-licenses": "npm-license > licenses.txt"
   },
   "devDependencies": {
     "@deephaven/eslint-config": "file:../eslint-config",
     "@deephaven/mocks": "file:../mocks",
     "@deephaven/prettier-config": "file:../prettier-config",
     "@deephaven/stylelint-config": "file:../stylelint-config",
-    "@deephaven/tsconfig": "file:../tsconfig"
+    "@deephaven/tsconfig": "file:../tsconfig",
+    "@vitejs/plugin-react": "^2.0.0",
+    "autoprefixer": "^10.4.8",
+    "postcss": "^8.4.14",
```

Comment:
Update the version in the root `package.json` as well (or you may be able to remove it entirely from there now).

Reply:
I can actually remove it from both b/c Vite includes postcss